ui/src/components/friends/add.rs

```diff
@@ -143,6 +143,25 @@ pub fn AddFriend(cx: Scope) -> Element {
                         // Use the default options for the remaining fields
                         ..Options::default()
                     },
+                    reset: request_sent.clone(),
+                    onreturn: move |_| {
+                        match DID::from_str(friend_input.get()) {
+                            Ok(did) => {
+                                if STATIC_ARGS.use_mock {
+                                    let mut ident = Identity::default();
+                                    ident.set_did_key(did);
+                                    state.write().mutate(Action::SendRequest(ident));
+                                    friend_input.set("".to_string());
```

Comment:
```suggestion
                                    friend_input.set(String::new());
```

Reply:
What might be better would be to clear the input outside of the if statement. Would reduce duplicated code